### ISSUE TITLE
Release 1.0

### ISSUE THREAD

**Issue by elharo** [googleapis/google-auth-library-java#183]
What, if anything, is standing in the way of a GA 1.0 release? 

Specifically I would like to remove this statement from the README:

"Note: This client is a work-in-progress, and may occasionally make backwards-incompatible changes."

What, if anything, still needs to be done before this can happen? This library is on the critical path for many other Google products. We need to finish it and commit to an API clients can rely on.

**Comment by garrettjonesgoogle** [comment]
Relevant best practice: https://github.com/GoogleCloudPlatform/cloud-opensource-java/blob/master/library-best-practices/JLBP-8.md "Advance widely used functionality to a stable version"

**Comment by elharo** [comment]
You can close this as a dupe of 101. I don't have permissions to do so.

**Comment by garrettjonesgoogle** [comment]
Duplicate of #101
